.github/workflows/synopsys.yaml

```diff
@@ -1,6 +1,7 @@
 name: Black Duck Policy Check
 on:
   pull_request:
```

Comment:
Due to how pull_request_target works (it uses the workflow from the target branch instead of the pull request), for this PR we need both pull_request and pull_request_target events so the pull_request can trigger the e2e tests as the pull_request_target is not present on the main branch prior to this PR. Once we merge this PR and pull_request_target is present on the main branch we will create a subsequent PR that will remove the pull_request event.